tests/test_wave_05.py

```diff
@@ -80,34 +85,56 @@ def test_create_goal(client):
     }
 
 
-@pytest.mark.skip(reason="test to be completed by student")
+# @pytest.mark.skip(reason="test to be completed by student")
 def test_update_goal(client, one_goal):
-    raise Exception("Complete test")
+    # raise Exception("Complete test")
     # Act
+    response = client.put("/goals/1", json={
+        "title": "Build a habit of going outside daily"
+    })
+    
     # ---- Complete Act Here ----
-
+    response_body = response.get_json()
     # Assert
     # ---- Complete Assertions Here ----
     # assertion 1 goes here
+    assert response.status_code == 200
     # assertion 2 goes here
+    assert "goal" in response_body
     # assertion 3 goes here
+    assert response_body == {
+        "goal": {
+            "id": 1,
+            "title": "Build a habit of going outside daily"
+        }
+    }
+    
     # ---- Complete Assertions Here ----
+    goal = Goal.query.get(1)
+    assert goal.title == "Build a habit of going outside daily"
```

Comment:
Good extra assertion to go and check that it was updated in the database and not just trust the response!